src/views/ProtestPage.js

```diff
@@ -61,9 +72,25 @@ function getSocialLinks(protest) {
   return items;
 }
 
+async function getDistance(protest) {
+  if (!protest || !protest.coordinates) {
+    return undefined;
+  }
+  try {
+    const position = await getCurrentPosition();
```

Comment:
You can get use the user coordinates from the `coordinates` component prop (https://github.com/guytepper/1km.co.il/blob/master/src/App.js#L236)